src/lib.rs

```diff
@@ -1892,6 +1893,129 @@ impl<'a> Builder<'a> {
         }
     }
 
+    // Check for unexpected namespace changes.
+    //
+    // The issue happens if developers added to the list of allowed tags any
+    // tag which is parsed in RCDATA state, PLAINTEXT state or RCDATA state,
```

Comment:
Is this supposed to say `RAWTEXT`, or just `RCDATA` and `PLAINTEXT`?

Reply:
Yeah, it should be RAWTEXT.